sljit_src/sljitNativeX86_common.c

```diff
@@ -518,7 +517,7 @@ static void get_cpu_features(void)
 	}
 
 	info[0] = 0x80000001;
-	info[2] = 0; /* Silences an incorrect compiler warning. */
+	info[2] = 0;
```

Comment:
Why the comment is removed? It is a bug in my compiler, it thinks the variable is never set (which is a false assumption).

Reply:
It was about it not being incorrect (under some circumstances), see the [pentium branch](https://github.com/carenas/sljit/tree/pentium) with one additional commit on top which might also do the initialization that was required.